network/src/protocols/identify/mod.rs

```diff
@@ -370,7 +370,18 @@ impl Callback for IdentifyCallback {
     }
 
     fn unregister(&self, context: &ProtocolContextMutRef) {
-        if context.session.ty.is_outbound() {
+        let protocol_version_match = self
+            .network_state
+            .with_peer_registry(|reg| {
+                reg.get_peer(context.session.id)
+                    .map(|p| p.protocol_version(context.proto_id))
+            })
+            .flatten()
+            .map(|version| version != "3")
```

Comment:
Is the meaning of '3' the same as `const LATEST_VERSION: &str = "3";` which is defined in `network/src/protocols/support_protocols.rs`?

Reply:
yes